ozaria/site/components/interactive/PageInteractive/draggableStatementCompletion/index.vue

```diff
@@ -72,58 +82,43 @@
         />
       </div>
 
-      <div class="art-container">
-        <img src="https://codecombat.com/images/pages/home/built_for_teachers1.png">
-      </div>
-    </div>
-
-    <div class="answer-row">
-      <statement-slot
-        v-for="(answerSlot, i) of answerSlots"
-        :key="i"
+      <div class="slot-row">
+        <statement-slot
+          v-for="(answerSlot, i) of answerSlots"
+          :key="i"
 
-        v-model="answerSlots[i]"
+          v-model="answerSlots[i]"
 
-        :draggable-group="draggableGroup"
+          :draggable-group="draggableGroup"
 
-        class="slot"
-      />
+          class="slot"
+          :label-text="(answerSlotLabels[i] || {}).text || ''"
```

Comment:
As far as I remember, we need to show the label text outside the label boxes, does this take care of that?

Reply:
Nope, will be handled in final UX step.  Focus here is functionality.